src/run.jl

```diff
@@ -299,6 +299,10 @@ function init_simulation()
     println("done")
   else
     print("Applying initial conditions... ")
+
+    # make sure that the random number generator is reseted and the ICs are reproducible in the julia REPL/interactive mode
+    seed!(0)
```

Comment:
Why is there no seed reset when loading a restart file? I know, I know, it would make more sense to store the current PRNG state in the restart files (that's for another day...), but at least that way we would also make restarts deterministic if they use the PRNG.